compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -1988,9 +1988,10 @@ object Types {
     }
 
     /** The argument corresponding to class type parameter `tparam` as seen from
-     *  prefix `pre`.
+     *  prefix `pre`. Can produce a TypeBounds type in case prefix is an & or | type
+     *  and parameter is non-variant.
      */
-    def argForParam(pre: Type)(implicit ctx: Context): Type = {
+    def argForParam(pre: Type, variance: Int)(implicit ctx: Context): Type = {
```

Comment:
Do we actually to pass the variance here? Shouldn't it always be the same as `symbol.paramVariance` ?

Reply:
It seems like we don't.